src/analysisd/decoders/security_configuration_assessment.c

```diff
@@ -735,7 +736,7 @@ static void HandleCheckEvent(Eventinfo *lf,int *socket,cJSON *event) {
 
     mdebug1("Checking event JSON fields");
 
-    if(!CheckEventJSON(event,&scan_id,&id,&name,&title,&description,&rationale,&remediation,&compliance,&check,&reference,&file,&directory,&process,&registry,&result,&status,&reason,&policy_id,&command,&rules)) {
+    if(!CheckEventJSON(event, &scan_id, &id, &name, &title, &description, &rationale, &remediation, &compliance, &condition, &check, &reference, &file, &directory, &process, &registry, &result, &status, &reason, &policy_id, &command, &rules)) {
```

Comment:
Split this into multiple lines like
![image](https://user-images.githubusercontent.com/1664307/64614510-f34fca00-d3d8-11e9-81c2-d2c5441126be.png)
In general try to keep lines ~140 max, but don't hesitate to keep them shorter if you have to split them anyway.

The same applies to many changed lines of this PR.